README.md

```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
 Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross-chain request and so on. Seeing is believing, there is a [15 minutes tutorial](https://docs.axonweb3.io/getting-started/for-dapp-devs/zero_to_axon_with_axon_cli) that will lead you to build your own chain and deploy a tiny application.
 
-### Native Cross-chain Communication
+### Native Cross-Chain Communication
 
 Openness and mobility are the foundation of social development, so is blockchain. Cross-chain function enhances liquidity for the web3 ecosystem. Axon develops native cross-chain communication without any bridge. Each Axon-based chain can speak to [CKB](https://www.nervos.org), other Axon-based chains, and any [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain protocols in the future.
 
+### Staking on Layer 1 CKB
+
+An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT tokens on CKB, which uses a Proof-of-Work (PoW) consensus mechanism. This unique staking design helps Axon-based sidechains enjoy the highest degree of decentralization and security from the layer 1 while maintain their high performance and sovereignty as independent layer 2 networks. 
+
 ## Roadmap
 
 Most of the infrastructure has been done and some substantial features to be developed are as below:
```

Comment:
Here are some substantial features we're working on:

Reply:
Sorry could you please elaborate?

Reply:
I suggest that deleting "Most of the infrastructure has been done", focusing on what we're doing to make Axon staking mechanism more outstanding/ some highlights of the staking mechanism.

Reply:
The Roadmap is another high-level section focusing on the project as a whole and not just staking I think. But thanks for bringing up a good point. @KaoImin do we need to add anything staking-related to roadmap?

Reply:
aha I see! but still, "Most of the infrastructure has been done" sounds pretty vague to me and make me wonder what exactly Axon has done. I guess it'd be better to explain a bit more on we've done, eg listing as bullet points below. And "most" often used to mean 60-90% of sth, it may brings some certain expectations like "wow Axon only needs a fewer days/months to get ready to launch"?